sdk/core/core-rest-pipeline/test/retryPolicy.spec.ts

```diff
@@ -363,42 +363,25 @@ describe("retryPolicy", function () {
     });
     await clock.runAllAsync();
     // should be one more than the default retry count
-    assert.strictEqual(next.callCount, DEFAULT_RETRY_POLICY_COUNT + 1);
+    assert.strictEqual(next.callCount, 1);
```

Comment:
These tests intended to check that logging happened. Logging in fact happened, and the logs indicated that retrying also happened, but at the time the retry policies where unified, it was harder to see how the retries shouldn’t have happened in this case.